draft-ietf-rtcweb-jsep.xml

```diff
@@ -1378,52 +1383,23 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-            <xref target="sec.pendingremotedescription" />) to null.
-            Any resources or candidates that were allocated by the
-            abandoned local description are discarded; any media that
-            is received will be processed according to the previous
-            local and remote descriptions. Rollback can only be used to
-            cancel proposed changes; there is no support for rolling
-            back from a stable state to a previous stable state. Note
-            that this implies that once the answerer has performed
-            setLocalDescription with his answer, this cannot be rolled
-            back.</t>
-
-            <t>A rollback will disassociate any RtpTransceivers that
-            were associated with m= sections by the application of the
-            rolled-back session description (see
-            <xref target="sec.applying-a-remote-desc" /> and
-            <xref target="sec.applying-a-local-desc" />). This means
-            that some RtpTransceivers that were previously associated
-            will no longer be associated with any m= section; in such
-            cases, the value of the RtpTransceiver's mid property MUST
-            be set to null, and the mapping between the transceiver and
-            its m= section index MUST be discarded. RtpTransceivers
-            that were created by applying a remote offer that was
-            subsequently rolled back MUST be stopped and removed from
-            the PeerConnection. However, a RtpTransceiver MUST NOT be
-            removed if a track was attached to the RtpTransceiver via
-            the addTrack method. This is so that an application may
-            call addTrack, then call setRemoteDescription with an
-            offer, then roll back that offer, then call createOffer and
-            have a m= section for the added track appear in the
-            generated offer.</t>
-
-            <t>A rollback is performed by supplying a session
-            description of type "rollback" with empty contents to
-            either setLocalDescription or setRemoteDescription,
-            depending on which was most recently used (i.e. if the new
-            offer was supplied to setLocalDescription, the rollback
-            should be done using setLocalDescription as well).</t>
+             <t>A rollback is performed by supplying a session
+             description of type "rollback" with empty contents to
+             either setLocalDescription or setRemoteDescription. It is
+             further described in <xref
+             target="sec.processing-a-rollback"/>.  Rollback can only
+             be used to cancel proposed changes; there is no support
+             for rolling back from a stable state to a previous stable
+             state. Note that this implies that once the answerer has
+             performed setLocalDescription with his answer, this
+             cannot be rolled back and neither the endpoint offerer or
```

Comment:
This sentence is awkward, and may need to change if we allow pranswer rollbacks.

Reply:
Note to self: this is covered.